speedtest, speedtest-cli: refresh (#5781)
tldr-pages/tldr: pages/common/speedtest.md

# speedtest

> Official command line interface for testing internet bandwidth using https://speedtest.net.

> > Note: some platforms link `speedtest` to `speedtest-cli`. If some of the examples in this page don't work, see `speedtest-cli`.

Reviewer: ```suggestion
> Note: some platforms link `speedtest` to `speedtest-cli`, which is not the official CLI. If some of the examples in this page don't work, see `speedtest-cli`.
```

Reviewer 2: I think is redundant with the first line of the command description.

Reviewer: If you guys think so, that's fine.  A bit of redunancy isn't always a problem, but the section is a little full as it is, so maybe we can skip it.

> > Note: some platforms link `speedtest` to `speedtest-cli`. If some of the examples in this page don't work, try `tldr speedtest-cli`.

Reviewer 3: Perhaps this will read better?
```suggestion
> Note: some platforms link `speedtest` to `speedtest-cli`. If some of the examples in this page don't work, see `speedtest-cli`.
```

> > Note: some platforms link `speedtest` to `speedtest-cli`.  If some of the examples in this page don't work, try `tldr speedtest-cli`.

Reviewer 2: ```suggestion
> Note: some platforms link `speedtest` to `speedtest-cli`. If some of the examples in this page don't work, try `tldr speedtest-cli`.
```
